2-mandatory/6-journey-planner.js

```diff
@@ -20,8 +20,8 @@
 function checkCodeIsThere(stringText) {
   let magicWord = "code";
   //edit code below
-  if (stringText) {
-    return stringText;
+  if (stringText.includes(magicWord)) {
+    return stringText.search(magicWord);
```

Comment:
you don't really need the include, if the magicWord can't be found the search returns -1

Reply:
thank you silly mistake